sandbox/lua/modules/graphite.lua

```diff
@@ -0,0 +1,135 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+Module contains utilities for Graphite
+--]]
+-- Imports
+require "string"
+require "math"
+require "table"
+
+local M = {}
+
```

Comment:
I knew I should have updated alert/annotation to use environments (see anomaly.lua).  https://github.com/mozilla-services/heka/issues/1496

It is too easy to leak into the global environment if you aren't careful (like thresh_bound and num_in_thresh below).